imagej/imagej-ops2/src/test/java/net/imagej/ops2/filter/NonLinearFiltersTest.java

```diff
@@ -161,7 +161,7 @@ public void testMinFilter() {
 	 */
 	@Test
 	public void testSigmaFilter() {
-		ops.op("filter.sigma").input(in, shape, oobFactory, 1.0, 0.0).output(out).compute();
+		ops.op("filter.sigma").input(in, shape, 1.0, 0.0, oobFactory).output(out).compute();
```

Comment:
Might want to test with/without the optional parameter.

Reply:
In general I had mixed feelings over testing the optional params. It is nice, though, to ensure we have things annotated correctly.